whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageService.java

```diff
@@ -53,7 +54,11 @@ public void verifyAndSetAsUndeliverable(final EmailMessageInfo message){
         }
 
         LOGGER.debug("Undeliverable message-id {} email {}", message.messageId(), StringUtils.join(message.emailAddresses(), ", "));
-        message.emailAddresses().forEach(email -> emailStatusDao.createEmailStatus(email, EmailStatus.UNDELIVERABLE));
+        try {
+            message.emailAddresses().forEach(email -> emailStatusDao.createEmailStatus(email, EmailStatus.UNDELIVERABLE));
+        } catch (DuplicateKeyException ex) {
+            LOGGER.warn("Email already exist in EmailStatus table {}", StringUtils.join(message.emailAddresses(), ", "), ex);
```

Comment:
Log at DEBUG or not at all, this is an expected situation and we don't need to take any action

Reply:
I turned into a debug